platform/mellanox/docker-syncd-mlnx.mk

```diff
@@ -18,3 +20,4 @@ $(DOCKER_SYNCD_BASE)_VERSION = 1.0.0
 $(DOCKER_SYNCD_BASE)_PACKAGE_NAME = syncd
 
 $(DOCKER_SYNCD_BASE)_RUN_OPT += -v /host/warmboot:/var/warmboot
+SONIC_BUSTER_DOCKERS += $(DOCKER_SYNCD_BASE)
```

Comment:
This sets syncd to build on buster only as MLNX-SAI is not yet ready to build on bullseye.

Reply:
is it because SDK or SAI itself not ready for 5.10?

Reply:
Yes SAI is not ready for 5.10 and the SDK is working on an official Debian 11 support release.